### DISCUSSION TITLE
Postgres 13 Deprecation Notice

### DISCUSSION THREAD

**Discussion by darora** [supabase/supabase#27946] [Changelog]
Supabase support for Postgres 13 is being deprecated as of 15th July 2024, and support for it will be fully removed on 15th November 2024.

All Postgres 13 projects should be upgraded to Postgres 15 before 15th November, 2024.

Any projects still on Postgres 13 after the 15th of November 2024 will be automatically upgraded to Postgres 15. If any Postgres extensions or functions are in use that cause the [upgrade process](https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects#pgupgrade) [to](https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects#extensions) [fail](https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects#objects-dependent-on-postgres-extensions), a backup will be taken instead, and the project will be paused.

Postgres 15 comes with numerous features, bug fixes and performance improvements. Check out the announcement blog posts to find out what each version introduces.

- [Postgres 14 Launch](https://supabase.com/blog/whats-new-in-postgres-14)
- [Postgres 15 Launch](https://supabase.com/blog/new-in-postgres-15)

## Deprecation Timeline

- **15th July**: All users are notified via email about Postgres 13 Deprecation. Users can self-serve upgrade to Postgres 15 [from our dashboard](https://supabase.com/dashboard/project/_/settings/infrastructure).
- **30th September**: Users are reminded of impending deprecation via email. 
- **30th October**: Users are sent a final email reminder.
- **15th November**: Any remaining projects on PG13 start getting migrated to PG15 (or paused, if there are upgrade issues).

**Comment by iampopal** [comment]
Will there be any braking changes if i Upgrade now, I am currently using version 13 of postgresql. and will there be any effects on storage?

**Reply by ans-4175** [reply]
```
{:message
 "Authentication error, reason: \"authentication query failed: %postgrex.error{message: nil, postgres: %{code: :insufficient_privilege, file: \\\"aclchk.c\\\", line: \\\"3650\\\", message: \\\"permission denied for view pg_shadow\\\", pg_code: \\\"42501\\\", routine: \\\"aclcheck_error\\\", severity: \\\"error\\\", unknown: \\\"error\\\", where: \\\"sql statement \\\\\\\"select usename::text, passwd::text from pg_catalog.pg_shadow\\\\n    where usename = p_usename\\\\\\\"\\\\npl/pgsql function pgbouncer.get_auth(text) line 5 at return query\\\"}, connection_id: 6993, query: nil}\"."}
 ```
 I got this error @darora. Any ideas on this?

**Reply by pcnc** [reply]
Hello @ans-4175. A fix has been recently deployed to our platform for the issue you've recently encountered.

If you're still experiencing this error, please contact support at [supabase.help](https://supabase.help) and we'll help address this for you and your projects.

**Reply by pcnc** [reply]
@iampopal there should be no effects on your project, except for a momentary unavailability on your project's end during the upgrade process' runtime.

**Comment by s-cavanh** [comment]
Hi @pcnc, I'm experiencing the same error as @ans-4175  on one of my production projects.
the data looks fine in the dashboard (tables section) but it gives same error in the SQL Editor.
i'm thinking it also affecting the access to Storage as well. (I cannot view any buckets)

**Reply by pcnc** [reply]
Hi @s-cavanh - if not done so already, please create a support ticket, including details related to your project, and we shall look into it.
